src/main/java/core/basesyntax/service/GenerateStringToWrite.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface GenerateStringToWrite {
+    String stringToWrite(Map<String, Integer> fruits);
```

Comment:
The method name should be a verb, understandable, and briefly describe what It does. 
FIx method name here